ci/nodepool/scripts/prepare_node.sh

```diff
@@ -14,12 +14,13 @@ source bootstrap.sh
 echo "Performaing a general update"
 sudo "${PKG_MGR}" update -y
 
-echo "Installing git"
-sudo "${PKG_MGR}" install -y git
-
 echo "Installing Puppet"
 if [ "${DISTRIBUTION}" == "redhat" ] && [ "${DISTRIBUTION_MAJOR_VERSION}" == "5" ]; then
-    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-5.noarch.rpm
+    sudo "${PKG_MGR}" install -y yum-security
+    curl -O https://dl.fedoraproject.org/pub/epel/epel-release-latest-5.noarch.rpm
```

Comment:
RHEL5 is not able to download and install an RPM, need to first download it and then install it.